src/pages/diffUrls.json

```diff
@@ -134,7 +134,8 @@
       "https://321naturelikefurfuroid.com/e/*",
       "https://449unceremoniousnasoseptal.com/e/*",
       "https://guidon40hyporadius9.com/e/*",
-      "https://www.turkanime.co/embed/*",
+      "https://cyamidpulverulence530.com/e/*",
+      "https://turkanime.co/embed/*",
```

Comment:
You have to keep the www. Here manually for now

Reply:
my rebase ended bad…